src/main/java/lotto/LottoReward.java

```diff
@@ -0,0 +1,16 @@
+package lotto;
+
+public enum LottoReward {
+
+    FIRST(2000000000), THIRD(1500000), FOURTH(50000), FIFTH(5000);
```

Comment:
```suggestion
    FIRST(2_000_000_000), THIRD(1500000), FOURTH(50000), FIFTH(5000);
```
언더 스코어를 활용하면 가독성을 살릴 수 있습니다 👍